### DISCUSSION TITLE
No desktop notifications

### DISCUSSION THREAD

**Discussion by ephraim** [mahmoudbahaa/outlook-for-linux#8] [General]
Hi there!

I use both teams-for-linux and outlook-for-linux. With Teams I get desktop notifications but for outlook not.

Any hints what I can do to debug why?

Cheers
Thilo

**Comment by simontunnat** [comment]
Same issue here but I guess this repo is dead?

**Comment by ChristoSnake** [comment]
Same for me.  I'd love it if this project could get some attention again!

**Comment by gj241** [comment]
Addressed in the maintained fork **outlook-for-linux v1.4.0**: https://github.com/gj241/outlook-for-linux/releases/tag/v1.4.0 — it now sends native desktop notifications via `notify-send` (with the `desktop-entry` hint so GNOME shows them properly), plus a new-mail sound. The project is active again. (Also relevant to #19.)
